byu-calendar-minimal-tile/byu-calendar-minimal-tile.scss

```diff
@@ -0,0 +1,53 @@
+$royal: #0057B8;
+$white: #FFFFFF;
+$medgray: #C5C5C5;
+$darkgray: #666;
+$black: #000;
+
+:host {
+  margin-bottom: 15px;
```

Comment:
Can you mixin-ify all of this file and have a byu-calendar-minimal-tile-common.scss file?